service_info/templates/cms/includes/menu.html

```diff
@@ -1,25 +1,86 @@
 {% load menu_tags i18n %}
 
-{% for child in children %}
-    <li class="{% if child.children %}parent{% endif %} {% if child.selected %}selected{% endif %}">
+<ul id="nav" class="hide-on-med-and-down">
+  {% for child in children %}
+    <li class="{% if child.selected %}active{% endif %}">
+      {% if child.children %}
+        <a class="dropdown-button" href="#!" data-activates="desktop-dropdown{{ forloop.counter0 }}">
+          {% if child.get_menu_title == "Blog" %}
+            <i class="material-icons left">description</i>
```

Comment:
SC-77 opened for adding a page extension where the material icon can be specified